### DISCUSSION TITLE
Separate notifications for different thresholds

### DISCUSSION THREAD

**Discussion by arielzusya** [alexjustesen/speedtest-tracker#1871] [Ideas]
I've seen a couple of others request conditional notifications (and I've added a +1 to those), but what I'm suggesting is a bit different. Namely to add the function of separate notifications for different thresholds (and if conditional notifications is added, to allow for separate notifications for different conditions). Here are some examples of what I was thinking:

all test results go to notification destination #1
notification 10% below expected bandwidth goes to notification destination #2
notification of 25% below expected bandwidth goes to notification destination #3
(and if conditional is added, bandwidth results X-% below expected threshold Y-times in a row go to destination #4)

Not a coder (wish I were so I could help), but love the project. Thank you! (Have a couple of other ideas so will post those next.)

**Comment by svenvg93** [comment]
What would be the use case for sending it to different destinations based on the threshold?

**Reply by arielzusya** [reply]
I imagine deployment in a business where a member of the IT team tasked with monitoring would receive the initial alert, then if speed issues progress, a higher level member (or perhaps supervisor) would also receive the alert, and so on. This would allow the rest of the IT team to not receive alerts unless the issue isn't addressed by the assigned team member and the issue continues to worsen. Haven't fully fleshed that out in my head yet but that's the sort of thing I was thinking.

**Reply by svenvg93** [reply]
I understand, the current threshold and notification systems are not designed to handle mulitple threshold with diffrent notification destinations. As well it might be complicated to build something like that 😅 . 
As well, sinces business are not really the target audience, I'm wondering if this will be used a lot by normal users. 

@alexjustesen What do you think.
